.goreleaser.yaml

```diff
@@ -0,0 +1,3 @@
+release:
```

Comment:
This change is unrelated

Reply:
@TylerHelmuth I realize that, It's the .goreleaser file - this branch got updated. Should it be taken out ?

Reply:
Yes

Reply:
Done